apps/prairielearn/src/pages/partials/question.ejs

```diff
@@ -177,37 +177,6 @@
           <%- include('submission', {submission: submission, submissionCount: submissions.length, submissionHtml: submissionHtmls[idx + MAX_TOP_RECENTS]}); %>
         <% }); %>
       </div>
-
-      <script>
-        $(function() {
-          $('.js-submission-body.render-pending').on('show.bs.collapse', function(e) {
-            const submissionId = $(this).data('submission-id');
-            // <% if (question_context === 'instructor' || question_context === 'public') { %>
-            const submissionContentUrl = `<%= urlPrefix %>/question/<%= question.id %>/preview/variant/<%= variant.id %>/submission/${submissionId}`;
-            // <% } else if (question_context === 'manual_grading') { %>
-            const submissionContentUrl = `<%= urlPrefix %>/assessment/<%= assessment.id %>/manual_grading/instance_question/<%= instance_question.id %>/variant/<%= variant.id %>/submission/${submissionId}`;
-            // <% } else { %>
-            const submissionContentUrl = `<%= urlPrefix %>/instance_question/<%= instance_question.id %>/variant/<%= variant.id %>/submission/${submissionId}`;
-            // <% } %>
-            const wasModalOpen = ($(`#submissionInfoModal-${submissionId}`).data('bs.modal') || {})._isShown;
-            $(`#submissionInfoModal-${submissionId}`).modal('hide');
-            $.get(submissionContentUrl, 'json')
-              .done((data) => {
-                $(`#submission-${submissionId}`).replaceWith($(data.submissionPanel).fadeIn('slow'));
-                MathJax.typesetPromise();
-              })
-              .fail(() => {
-                $(`#submission-${submissionId}-body`).html('<div class="card-body submission-body">Error retrieving submission</div>');
-              })
-              .always(() => {
-                // Restore modal state if need be
-                if (wasModalOpen) {
-                  $(`#submissionInfoModal-${submissionId}`).modal('show');
-                }
-              });
-          });
-        });
-        </script>
```

Comment:
This code was moved to the question.ts asset, and converted to use vanilla JS (where possible) and fetch.